CHANGELOG.md

```diff
@@ -117,6 +139,40 @@
   - [changelog](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#8200)
   - [diff](https://github.com/getsentry/sentry-cocoa/compare/8.19.0...8.20.0)
 
+## 8.0.0-beta.2
```

Comment:
This is at the wrong place in the changelog

Reply:
What do you mean?

Reply:
The v8 things should be at the top, right? That's not the case right now.

Reply:
We discussed this and we'll keep the release order so beta should be fine where it is now